Только в mitter-0.4.5/mitterlib: __init__.pyc
Только в mitter-0.4.5/mitterlib: threadhttp.pyc
Только в mitter-0.4.5/mitterlib: twitter.pyc
diff -rup mitter-0.4.5_orig/mitterlib/ui/ui_pygtk.py mitter-0.4.5/mitterlib/ui/ui_pygtk.py
--- mitter-0.4.5_orig/mitterlib/ui/ui_pygtk.py	2009-04-15 13:10:19.000000000 +0300
+++ mitter-0.4.5/mitterlib/ui/ui_pygtk.py	2009-04-19 15:27:35.000000000 +0300
@@ -477,11 +477,8 @@ class Interface(object):
         info_area = gtk.HBox(True, 0)
 
         self.char_count = gtk.Label()
-        info_area.pack_start(gtk.Label('What are you doing?'))
-        info_area.pack_start(self.char_count)
 
         text_area = gtk.VBox(True, 0)
-        text_area.pack_start(info_area)
         text_area.pack_start(self.update_text)
 
         update_button = gtk.Button(stock=gtk.STOCK_ADD)
@@ -490,6 +487,8 @@ class Interface(object):
         update_box = gtk.HBox(False, 0)
         update_box.pack_start(text_area, expand=True, fill=True,
                 padding=0)
+        update_box.pack_start(self.char_count, expand=False, fill=False,
+                padding=0)
         update_box.pack_start(update_button, expand=False, fill=False,
                 padding=0)
 
